### DISCUSSION TITLE
Call to undefined function powerGridJsFramework()

### DISCUSSION THREAD

**Discussion by kenmdavids** [Power-Components/livewire-powergrid#474] [Q&A]
Thanks for the great plugin.
I get an issue **(Call to undefined function powerGridJsFramework())**
the moment I add the @powerGridScripts in the body. 
I am using Laravel v9 and PHP 8.0.15
I think the issue is coming from scripts in the published assets. 

`@if(powerGridJsFramework() === JS_FRAMEWORK_ALPINE)
    <script src="{{ config('livewire-powergrid.js_framework_cdn.alpinejs') }}" defer></script>
@endif`

Any quick help will be appreciated.

**Comment by luanfreitasdev** [comment]
Hello, this is no longer in the current version. What version are you using?

**Reply by kenmdavids** [reply]
am using v3.1.0.
now it even returns 
`Undefined variable $exportOption`

**Reply by luanfreitasdev** [reply]
It's probably getting it from the vendor from resources.

**Comment by kenmdavids** [comment]
<img width="912" alt="Screenshot 2022-08-02 102133" src="https://user-images.githubusercontent.com/10721732/182316415-5c076818-3f2f-4238-ba90-97d56387126d.png">

**Reply by kenmdavids** [reply]
how do I clear it from the vendors
